crates/core/src/yarn_fn/function_wrapping.rs

```diff
@@ -22,6 +22,11 @@ use variadics_please::all_tuples;
 ///   - A numeric type, i.e. one of [`f32`], [`f64`], [`i8`], [`i16`], [`i32`], [`i64`], [`i128`], [`u8`], [`u16`], [`u32`], [`u64`], [`u128`], [`usize`], [`isize`]
 ///   - [`String`]
 ///
+/// If the `bevy` feature is active then it is also possible to register a Bevy `System` and call it from Yarn. The `System` will receive the parameters passed to the yarn
+/// as it's input. The `System`'s input must adhere to the same rules as given above for regular function parameters. `System`s cannot currently take tuples of values in
+/// their input but that limitation will be fixed soon. The `System`'s output must adhere to the rules for a regular function's return type listed above.
+/// The `System` may take any `SystemParam`.
+///
```

Comment:
Some thoughts:
- Could we omit the part about "will be fixed soon"? I prefer the documentation to be timeless.
- Could we add an example for this? You can conditionally include documentation based on a feature, see e.g. how `avian` uses different examples for the 3d and 2d variants.

Reply:
If you don't have time, this is non-blocking :)

Reply:
Tweaked the documentation and added a simple example. I didn't feature gate it because it seems like it's worth having in the general docs to let you know what you can do with the feature. And I believe the docs will always be built with --all-features? But if it needs to be conditional or you feel like it's clearer to have it hidden in versions of the docs that don't activate the feature I can do that.